core/commands/add.go

```diff
@@ -120,6 +123,8 @@ You can now check what blocks have been created by:
 		cmdkit.BoolOption(fstoreCacheOptionName, "Check the filestore for pre-existing blocks. (experimental)"),
 		cmdkit.IntOption(cidVersionOptionName, "CID version. Defaults to 0 unless an option that depends on CIDv1 is passed. (experimental)"),
 		cmdkit.StringOption(hashOptionName, "Hash function to use. Implies CIDv1 if not sha2-256. (experimental)").WithDefault("sha2-256"),
+		cmdkit.BoolOption(inlineOptionName, "Inline small blocks into CIDs. (experimental)"),
+		cmdkit.IntOption(inlineLimitOptionName, "Maximum block size to inline. (experimental)").WithDefault(64),
```

Comment:
I would like to know the reasons for this.  Is it the change in the API string because of the use of WithDefault() or something else?

Reply:
The reasoning is simply "fewer knobs" (or "be conservative"). That is, we don't really *need* this knob so we might as well add it later when needed (when a user presents a use-case for changing the default).

Reply:
I don't agree with this philosophy but won't object strongly.

I can see a case of wanting either 32 or 64 and don't think there is a clear answer.  I am okay with limiting it to either 32 or 64 if you think that will be better.

Reply:
User here looking forward to this feature, I'm mirroring a large-ish dataset using filestore to avoid disk duplication, in this case there is a few files being updated often which is causing issues with missing data, these are 41-42 bytes in size. example: http://distfiles.gentoo.org/experimental/timestamp.x 
So minimal size here is 64 bytes - but I don't see any good reason to not add the option for the user to change this. Not adding this option could also result in assumptions being made about the actual max of ident hashes.

Reply:
The issue is that we *may* actually need to define a limit on the maximum CID size and 64 bytes is *really* long (see my comment below). However, having to define a new block for a ~40 byte file really does suck.

Let's wait for the browser working group to comment.

Reply:
Yes 64 bytes is long, but if it is part of a directory entry, the user is ever unlikely to need to actual CID and it won't show up as part of the URL.  All the more reason to keep this option.

I could change the default to 32 though so it won't create a problem.  When someone knows what they are doing they can increase it to suit there needs.

Reply:
Unless a user just adds a file directly. E.g., `/ipfs/z3Fxd5vkFxAu4YbpXLBj8EvUCEeXLhNRGaYpEL43EHt99kaB82LfGbEVBqUJPaSZsHtJANxV8EGDZ`

Reply:
We still need to talk to the browser people. I know there were talks about supporting `https://CID.dweb.link` and that won't work for long CIDs. The current plan is `ipfs://CID` and there may be length restrictions.